Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,132 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+
+namespace Fuse.Controls
+{
+	/**
+		TODO write docs please
+	*/
+	public class SystemScreenConfigBase : Behavior
+	{
+		public enum Visibility
+	    {
+	        None,
+	        Minimal,
+	        Full,
+	    }
+
+	    private static List<SystemScreenConfigBase> rootedConfigs = new List<SystemScreenConfigBase>();
+
+	    protected Timer _timer;
+		private bool _rooted;
+
+	    protected override void OnRooted()
+		{
+			base.OnRooted();
+
+			_rooted = true;
```

Comment:
This variable shoudl not be needed. Node's expose a `IsRootingStarted` and `IsRootingComplete`.